Clarify exact date for requiring Python 3 support (#132)
astropy/astropy-procedures: documents/affiliated_package_review_guidelines.md

<table>
<tr>
<td width=150><img src="https://img.shields.io/badge/Red-red.svg" alt="Red"></td>

> <td>Nothing is 'unacceptable' at this point, but once after 1 January 2020 ‘Only compatible with Python 2’ goes from orange to red.</td>

Reviewer: ```suggestion
<td>Nothing is currently 'unacceptable'. Starting 1 January 2020, ‘Only compatible with Python 2’ will be red.</td>
```

Reviewer: Also add on line 171 what Orange will change to: <td>Only compatible with Python 2. Starting 1 January 2020: 'Not fully Python 3'</td>. (I just made up the text...I'm not sure what Orange should be.)

Reviewer 2: Maybe we should take the approach of astropy core and effectively remove all references to Py2 (as if it never existed).  So the words here would be simply "Compatible with Python 3" or "Not compatible with Python 3".

Reviewer 2: Leading to:
```
<td>Nothing is currently 'unacceptable'. Starting 1 January 2020, 
    ‘Not compatible with Python 3’ will be red.</td>
```
On a different note, is the weird (non-ASCII?) single quote in the second bit intended?

Author: @taldcroft @kelle - thanks for the suggestions! I've gone with @taldcroft's latest suggestion. The non-ASCII quote has been removed.
